service/web/publishserver_test.go

```diff
@@ -130,8 +131,10 @@ func TestPublishServer_PublishSuccess(t *testing.T) {
 			},
 		},
 	}
+	cacheMap := make(map[string]bool)
+	cacheMap["projects/project123/topics/test-topic"] = true
+	tasks.SetCacheData(cacheMap)
```

Comment:
Can we avoid explicitly setting cache data? We want to test cache being populated by Consul watch